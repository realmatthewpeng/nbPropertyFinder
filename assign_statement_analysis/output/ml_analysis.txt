Total number of lines found: 1

modeling_df = pd.read_csv('data/weather_with_avg_hourly_flight_delay.csv', index_col=False)

Incorrect Lines:


Precision = 1.0
Recall = 1.0

