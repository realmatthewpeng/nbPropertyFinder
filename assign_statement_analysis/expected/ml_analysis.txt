modeling_df = pd.read_csv('data/weather_with_avg_hourly_flight_delay.csv', index_col=False)